### PR TITLE
Adds the base for cargo bounties

### PR DESCRIPTION
**Pull request by Aranclanos**
### Purpose
Adds cargo bounties base, three different bounties for cargo techs to complete from roundstart.
Moves the CargoData SO and script to their correct folders.
Once arriving centcomm the cargo shuttle will check the contents of each wrapped objects for processing and ignore the wrapping object price values.
Adds the bounty window to the cargo console UI.
Reworks the cargo console UI, making it simpler and easier to understand and fixing some bugs when two consoles were used at the same time.
Adds the base for the emagging of the cargo console.

makes progress on #6150.

### Notes:
To check for completion status of the bounties, they'll check for itemattributes traits
The three base bounties are to get 5 wrenches, 5 welders and 5 crowbars. They reward 200 points.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Managers/CargoManager.cs [hunk 1]
```diff
@@ -303,18 +308,66 @@ public void DestroyItem(ObjectBehaviour item, HashSet<GameObject> alreadySold)
 			export.Count += count;
 			export.TotalValue += credits;
 
+			var itemAttributes = item.GetComponent<ItemAttributesV2>();
+			if (itemAttributes)
+			{
+				foreach (var itemTrait in itemAttributes.GetTraits())
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/CargoManager.cs:314]: can you reformulate this logic to reduce nesting? Maybe use ``Validations.HasItemTrait`` instead of iterating all traits or even extract some portions into private methods.

File: UnityProject/Assets/Scripts/Systems/Cargo/CargoData.cs [hunk 2]
```diff
@@ -6,8 +6,8 @@ namespace Systems.Cargo
 {
 	public class CargoData : ScriptableObject
 	{
-		[ReorderableList]
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Systems/Cargo/CargoData.cs:9]: why did you remove the tooltip and reorderable attribute?

**Comment by Aranclanos** [comment]
updated

File: UnityProject/Assets/Scripts/Managers/CargoManager.cs [hunk 3]
```diff
@@ -223,11 +219,20 @@ public void OnShuttleArrival()
...
+			OnShuttleUpdate.Invoke();
 		}
 
-		public void DestroyItem(ObjectBehaviour item, HashSet<GameObject> alreadySold)
+		public void ProcessCargo(ObjectBehaviour item, HashSet<GameObject> alreadySold)
 		{
+			var wrappedItem = item.GetComponent<WrappedBase>();
+			if (wrappedItem)
+			{
+				var wrappedContents = wrappedItem.GetOrGenerateContent().GetComponent<ObjectBehaviour>();
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/CargoManager.cs]: this ObjectBehaviour can be null

**Comment by PerfectTangent** [comment]
the bounties should appear in the status tab under the centcom messages whenever they are fulfilled.

**Comment by PerfectTangent** [comment]
the bounties themselves need to have an entry for the name of the bounty and a separate entry that can be filled with flavor text. Since the flavor text can get pretty long for some bounties, I think the description should be viewed by clicking the bounty listing in the bounty tab; so, a bounty would display in the tab as `200 credits - send 10 pens` and when you click on the bounty, it would expand to display the full description: `200 credits - send 10 pens - We are hosting the intergalactic pen balancing competition. We need you to send us some standardized ball point pens.`

Since bounties can be filled out over multiple shipments, I think we need to see the progress made on a bounty. If you have to ship 5 wrenches and you ship 2 wrenches, it should show in the bounty 2/5 wrenches sent.
